README.md

```diff
@@ -1,7 +1,8 @@
 Iris
 ====
 
-[![Build Status](https://secure.travis-ci.org/SciTools/iris.png)](http://travis-ci.org/SciTools/iris)
+[![Build Status](https://api.travis-ci.org/repositories/SciTools/iris.svg?branch=master)](http://travis-ci.org/SciTools/iris/branches)
+[![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.51860.svg)](http://dx.doi.org/10.5281/zenodo.51860)
 
 (C) British Crown Copyright 2010 - 2015, Met Office
```

Comment:
@ajdawson looks like the copyright dates could _definitely_ do with being updated now!